packages/connector-naver/src/types.ts

```diff
@@ -0,0 +1,34 @@
+import { z } from 'zod';
+
+export const naverConfigGuard = z.object({
+  clientId: z.string(),
+  clientSecret: z.string(),
+});
+
+export type naverConfig = z.infer<typeof naverConfigGuard>;
+
+export const accessTokenResponseGuard = z.object({
+  access_token: z.string(),
+  scope: z.string().optional(),
+  token_type: z.string(),
+});
+
+export type AccessTokenResponse = z.infer<typeof accessTokenResponseGuard>;
+
+export const userInfoResponseGuard = z.object({
```

Comment:
would you mind posting the reference of the user info profile response here, it took me a while to find the right place. Maybe it can also be a little bit of trouble for other developers.

https://developers.naver.com/docs/login/profile/profile.md
https://developers.naver.com/docs/login/devguide/devguide.md

Reply:
Thats a good idea. I've added.